server/tournaments/index.ts

```diff
@@ -1248,7 +1248,13 @@ const commands: ChatCommands = {
 			this.checkCan('tournaments', null, room);
 			const [format, generator, cap, mod, name] = target.split(',').map(item => item.trim());
 			if (!target || !format || !generator) {
-				return this.sendReply(`Usage: /tour ${cmd} <format>, <type> [, <comma-separated arguments>]`);
+				if (room.game) {
+					this.errorReply(`You cannot have a tournament until the current room activity is over: ${room.game.title}`);
+					return;
```

Comment:
```suggestion
					throw new Chat.ErrorMessage(`You cannot start a tournament until the current room activity (${room.game.title}) ends.`);
```

Reply:
I don’t recognize that new API. What does it do?

Reply:
It throws an error that when the command parser catches, it returns the error's `message` in an `errorReply`.

Reply:
I just copied this from elsewhere in the file. Should that spot be changed as well?

Reply:
Probably yes.

Reply:
This file is actually chock-full of the old `errorReply` syntax, so that should be a separate PR.